docs/concepts/realms.md

```diff
@@ -4,18 +4,41 @@ id: realms
 
 # Realms
 
-A realm refers to a specific instance of a smart contract that can be written
-in [Gno](./gno-language.md). The potentials of realms are endless - you can create virtually any
-application in your mind with built-in composability,
-transparency, and censorship resistance. Here are some ideas of what you can build with realms:
+A Realm represents an on-chain entity which has an address, can execute code
+and is capable of managing Coins associated with the given address.
+It boils down to one of two kinds of realms:
 
-* Self-custodial financial exchanges (decentralized exchanges).
-* Lending platforms with better rates.
-* Transparent insurance systems.
-* Fair and accessible voting systems.
-* Logistics and supply chain networks.
+- User realms: an end-user of the blockchain, who has a private key from
+    which the public key and address are subsequently derived.
+- Code realms: a package published on-chain, with storage through globlal
+    variables and access to the Banker. It is Gno's specific flavour of a
+    "smart contract".
 
-## Packages vs Realms
+In many instances, when simply saying a "realm", the latter is intended;
+however, users are also considered "realms".
+
+Code realms offer limitless potential - they allow you to create virtually any
+application while being:
+
+- Composable: using other realms is an `import` statement away.
+- Transparent: like all code on Gno.land, they must be added to the chain in
```

Comment:
```suggestion
- Transparent: like all code on gno.land, they must be added to the chain in
```
I understand we are now supposed to refer to gno.land in all lowercase from now on.